src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -740,7 +661,7 @@ PhasePartition(ts::PhaseNonEquil) = ts.q
 """
     saturation_adjustment(e_int, ρ, q_tot)
 
-Compute the temperature that is consistent with
+Compute the temperature `T` that is consistent with
```

Comment:
How about adding after the definition of the argument (l. 669) something like ``by finding the root of internal_energy_sat(T, \rho, q_tot) - e_int = 0``?

Reply:
Implemented in #566